src/content/en/updates/2017/05/devtools-release-notes.md

```diff
@@ -134,6 +262,51 @@ highlights the jumpable destinations in blue.
   </figcaption>
 </figure>
 
+See [Get Started With Debugging JavaScript][GSDJ] to learn the basics of
+debugging in DevTools.
+
+[GSDJ]: /web/tools/chrome-devtools/javascript/
+
+{% framebox width="auto" height="auto" enable_widgets="true" %}
+<script>
+var like = 'Great. You\'re welcome to tweet us at ' +
+    '<a href="https://twitter.com/chromedevtools">@ChromeDevTools</a> or start a thread in the ' +
+    '<a href="https://groups.google.com/forum/#!forum/google-chrome-developer-tools">mailing ' +
+    'list</a> if you\'ve got any ideas on how to improve this.';
+var dislike = 'Sorry to hear that. Please tweet us at ' +
+    '<a href="https://twitter.com/chromedevtools">@ChromeDevTools</a> or start a thread in the ' +
+    '<a href="https://groups.google.com/forum/#!forum/google-chrome-developer-tools">mailing ' +
+    'list</a> and let us know what\'s wrong with it.';
+var label = 'WNDT60 / Continue To Here / Sentiment';
+var feedback = {
+  "category": "DevTools",
+  "question": 'What do you think of this change?',
+  "choices": [
+    {
+      "button": {
+        "text": "I Like It"
+      },
+      "response": like,
+      "analytics": {
+        "label": label
+      }
+    },
+    {
+      "button": {
+        "text": "I Don't Like It"
+      },
+      "response": dislike,
+      "analytics": {
+        "label": label,
+        "value": 0
+      }
+    }
+  ]
+};
+</script>
+{% include "web/_shared/multichoice.html" %}
+{% endframebox %}
```

Comment:
This is the 3rd time I've seen this code. Can this be pulled into an external file and included where needed?